package.json

```diff
@@ -37,11 +37,11 @@
     "pify": "^2.2.0",
     "rimraf": "^2.4.3",
     "run-applescript": "^2.0.0",
-    "xdg-trashdir": "^2.0.0"
+    "xdg-trashdir": "^2.1.0",
+    "path-exists": "^2.0.0"
```

Comment:
I'm sure (or at least I hope) you have better things to do, than manually sorting lines in text files...
I know I have :-)